Nosach/web-quiz-project/src/main/java/controllers/QuizController.java

```diff
@@ -49,7 +80,7 @@ public String showQuizAddingForm(Model model){
     }
 
     @RequestMapping(method = POST, value = "add")
-    public String addQuizBasicInfo(HttpSession session, WebRequest req){
+    public String addQuizBasicInfo(HttpSession session, WebRequest req, Principal principal){
```

Comment:
Do you need this session now?

Reply:
I'm using session attribute in this method. I can change session to @SessionAttribute.
Will fix this in next commit.